android-auto-app/build.gradle.kts

```diff
@@ -49,7 +49,8 @@ dependencies {
 
   // Please review the compatibility guide. This app is showcasing the latest features.
   // https://github.com/mapbox/mapbox-maps-android/tree/main/extension-androidauto#compatibility-with-maps-sdk-v10
-  implementation("com.mapbox.maps:android:10.9.0-beta.2")
+//  implementation("com.mapbox.maps:android:10.9.0-beta.2")
```

Comment:
maybe remove it?

Reply:
It will be reverted, this change is to validate it doesn't break existing Android Auto code.